.github/actions/system-test/action.yml

```
@@ -123,7 +123,11 @@ runs:
         export MICROCLOUD_SNAP_CHANNEL="${{ matrix.microcloud }}"
 
         cd test
-        sudo --preserve-env=GOCOVERDIR,DEBUG,GITHUB_ACTIONS,MICROCLOUD_DEBUG_PATH,MICROCLOUDD_DEBUG_PATH,SKIP_VM_LAUNCH,SNAPSHOT_RESTORE,TEST_STORAGE_SOURCE,TESTBED_READY,BASE_OS,LXD_SNAP_CHANNEL,MICROCEPH_SNAP_CHANNEL,MICROOVN_SNAP_CHANNEL,MICROCLOUD_SNAP_CHANNEL ./main.sh setup
+        if [ "${{ matrix.suite}}" = "terraform_demo" ]; then
```

This GHA syntax is close to `bash` but not quite. I think the surrounding spaces are needed:
```suggestion
        if [ "${{ matrix.suite }}" = "terraform_demo" ]; then
```